modules/host/contractmanager/persist.go

```diff
@@ -31,6 +32,34 @@ type (
 	}
 )
 
+func (sf *savedStorageFolder) Equals(sfb savedStorageFolder) bool {
```

Comment:
this should probably be inlined into `savedSettings.Equals`. It's not likely to be useful on its own.